change/react-native-windows-2020-04-24-19-55-04-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```

Comment:
Would you be able to manually change "prerelease" to "patch"? Beachball has a bug where it will otherwise no-op in this branch.

Reply:
Ok, done